examples/practice-typescript/src/scripts/index.tsx

```diff
@@ -1,11 +1,13 @@
 import { createRoot } from 'react-dom/client';
 import { StrictMode } from 'react';
 import Logo from 'components/Logo/Logo';
+import SideBarMenu from 'components/SideBarMenu/SideBarMenu';
 
 const domNode = document.getElementById('root');
 const root = createRoot(domNode);
 root.render(
   <StrictMode>
     <Logo />
+    <SideBarMenu active="Product management" />
```

Comment:
Split this into `App.tsx` this file should not grow

Reply:
Fixed